src/Microsoft.DotNet.Watcher.Tools/DotNetWatchCommon.targets

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="_WriteGeneratedWatchList" >
+    <RemoveDuplicates Inputs="@(Watch)">
+        <Output TaskParameter="Filtered" ItemName="_WatchFiltered" />
+    </RemoveDuplicates>
+    <Message Text="@(_WatchFiltered, '%0A')"/>
```

Comment:
Is this needed for the error message in case in case `dotnet msbuild` errors out?